cmd/catalyst-node/catalyst-node.go

```diff
@@ -478,11 +497,84 @@ func redirectHandler(redirectPrefixes []string, nodeHost string) http.Handler {
 
 		rPath := fmt.Sprintf(pathTmpl, fullPlaybackID)
 		rURL := fmt.Sprintf("%s://%s%s", protocol(r), bestNode, rPath)
+		rURL, err = resolveNodeURL(rURL)
```

Comment:
Why do we need to resolve the node here? Didn't we already query Mist for the best node and it's already resolved?

Reply:
MistUtilLoad assumes every node is accessible on 443, which isn't true in our staging or canary environments, so we can use Serf ot override here